doc/decisions/decision_process.md

```diff
@@ -24,24 +26,42 @@ Substantial decisions must be made in a transparent and participative way.
   - Rebase only if the decision was already accepted and has a merge conflict.
 - For reviewers:
   - Prefer to directly give suggestions how to change sentences.
-  - General questions should be asked in the root of "Conversation" and not at random sentences in the review.
+  - General questions should be asked in the root of "Conversation" and not at vaguely related sentences in the review.
+- Decision PRs do not significantly change anything but one decision.
+- Changes not changing the decision step or the direction of the decision are not decision PRs.
+- The person merging the decision PR must be someone else as the person that created the decision.
+- The purpose of decisions is to have clear descriptions of technical problems and solutions.
+  There is no claim that decisions contain everything that was said.
+  In particular corrections of wrong decision text is, if at all, only visible via git history.
+  Rather it is important that decisions:
+  - contain everything relevant, and
+  - what is written is technically correct.
+- Participants are explicitly allowed to skip discussions and only read the decisions text.
+  If the problem is not yet clear, only partial reviews are encouraged.
+  @markus2330 will do so of time budget reasons and to make sure the decisions stay understandable.
```

Comment:
```suggestion
- Reviewers are only required to read the files in the PR.
  They may ignore the discussions surrounding the decision.
  Reviews should focus on the "Problem" section first, and only when that is agreed upon focus in the other parts.
  It is encouraged that at least one reviewer provides a review _without_ participating in the discussion.
  This ensures that there aren't any unintentional shared assumptions between discussion participants.
```

1. "clear" is not precise enough for me. "Clear" to who? I think agreement is a much better condition. This also leaves the option for reviewers to say "I don't fully understand it, but I can see that this could be a problem".
2. I don't think we should mention individual people in the decision.